internal/google/sheets/models.go

```diff
@@ -135,12 +135,7 @@ func (r rawQueryRowsResult) convertRawValue(cellIdx int, cell rawQueryRowsResult
 	case "boolean":
 		return strings.ToLower(cell.Raw) == "true", nil
```

Comment:
Should we use `cell.Value `instead? 

If yes, we can remove `cell.Raw` altogether.

Reply:
Yes, this one we can remove. How about the datetime btw? I think we can remove the entire type check actually since we would not be able to inject such data type anyway, right? It will be converted into the prefix single quote string internally.

Reply:
> How about the datetime btw? I think we can remove the entire type check actually since we would not be able to inject such data type anyway, right? 

Yup, there's no way for us to insert a date to gsheet. I think can remoev.